pkg/reconciler/revision/revision_test.go

```diff
@@ -488,118 +489,62 @@ func TestNoQueueSidecarImageUpdateFail(t *testing.T) {
 }
 
 func TestGlobalResyncOnConfigMapUpdateRevision(t *testing.T) {
-	// Test that changes to the ConfigMap result in the desired changes on an existing
-	// revision.
-	tests := []struct {
-		name              string
-		configMapToUpdate *corev1.ConfigMap
-		callback          func(*testing.T) func(runtime.Object) HookResult
-	}{{
-		name: "Update LoggingURL", // Should update LogURL on revision
-		configMapToUpdate: &corev1.ConfigMap{
-			ObjectMeta: metav1.ObjectMeta{
-				Namespace: system.Namespace(),
-				Name:      metrics.ConfigMapName(),
-			},
-			Data: map[string]string{
-				"logging.enable-var-log-collection": "true",
-				"logging.revision-url-template":     "http://log-here.test.com?filter=${REVISION_UID}",
-			},
-		},
-		callback: func(t *testing.T) func(runtime.Object) HookResult {
-			return func(obj runtime.Object) HookResult {
-				revision := obj.(*v1.Revision)
-				t.Logf("Revision updated: %v", revision.Name)
-
-				const expected = "http://log-here.test.com?filter="
-				got := revision.Status.LogURL
-				if strings.HasPrefix(got, expected) {
-					return HookComplete
-				}
-
-				t.Logf("No update occurred; expected: %s got: %s", expected, got)
-				return HookIncomplete
-			}
-		},
-	}, {
-		name: "Update ContainerConcurrency", // Should update ContainerConcurrency on revision spec
-		configMapToUpdate: &corev1.ConfigMap{
-			ObjectMeta: metav1.ObjectMeta{
-				Namespace: system.Namespace(),
-				Name:      config.DefaultsConfigName,
-			},
-			Data: map[string]string{
-				"container-concurrency": "3",
-			},
-		},
-		callback: func(t *testing.T) func(runtime.Object) HookResult {
-			return func(obj runtime.Object) HookResult {
-				revision := obj.(*v1.Revision)
-				t.Logf("Revision updated: %v", revision.Name)
-
-				expected := int64(3)
-				got := *(revision.Spec.ContainerConcurrency)
-				if got != expected {
```

Comment:
This test never worked properly because this should be `==` to check what it is intending to check.

I'm ok with removing it, since it effectively tests nothing today, but we should address the coverage gap we have here in a follow up.